libcontainer/nsenter/nsexec.c

```diff
@@ -201,7 +207,8 @@ static void update_setgroups(int pid, enum policy_t setgroup)
 		 * open(2) or write(2) will return ENOENT. This is fine.
 		 */
 		if (errno != ENOENT)
-			bail("failed to write '%s' to /proc/%d/setgroups", policy, pid);
+			bail("failed to write '%s' to /proc/%d/setgroups",
+			     policy, pid);
```

Comment:
harder to see what's new when the diff includes lots of odd extra spaces and new lines throughout